src/openrct2/windows/ride.c

```diff
@@ -4028,7 +4045,10 @@ static void window_ride_maintenance_paint(rct_window *w, rct_drawpixelinfo *dpi)
 		if (stringId != 0) {
 			if (stringId == STR_CALLING_MECHANIC) {
 				gfx_draw_string_left_wrapped(dpi, NULL, x + 4, y, 280, stringId, COLOUR_BLACK);
-			} else {
+			} 
+			else if(stringId == STR_NO_MECHANICS_ARE_HIRED_MESSAGE){
```

Comment:
You can handle this condition in the first `if` block. No need for an extra `else if`.